pages/collector_sidecar.rst

```diff
@@ -280,13 +280,19 @@ First start
 Once you installed the Sidecar package you are ready to start the service for the first time. Decide which backend you want to use. Enable or disable the single
 backends by setting ``enabled: true`` or respectively to ``false``. Now start the Sidecar, depending on your operating system you can do this with:
 
-+---------------+---------------------------------------------------------------------------------------------+
-| Debian/Ubuntu | ``sudo start collector-sidecar``                                                            |
-+---------------+---------------------------------------------------------------------------------------------+
-| RedHat/CentOS | ``sudo systemctl start collector-sidecar``                                                  |
-+---------------+---------------------------------------------------------------------------------------------+
-| Windows       | ``C:\Program Files\graylog\collector-sidecar\graylog-collector-sidecar.exe -service start`` |
-+---------------+---------------------------------------------------------------------------------------------+
++-------------+----------------------------------------------------------------------------------------------+
+| SystemD     | ``sudo systemctl start collector-sidecar``                                                   |
++-------------+----------------------------------------------------------------------------------------------+
+| SysV        | ``sudo start collector-sidecar``                                                             |
++-------------+----------------------------------------------------------------------------------------------+
+| Windows     | ``C:\Program Files\graylog\collector-sidecar\graylog-collector-sidecar.exe -service start``  |
++-------------+----------------------------------------------------------------------------------------------+
+
+If you're unsure which initsystem your distro uses try this one liner::
+
+    for initsys in systemd upstart init; do ps x | grep -qi "$initsys" && echo $initsys && break; done
```

Comment:
I'd rather use the following command, which shows the command of PID 1:
```text
# ps -h -o comm -p 1
```

Reply:
agreed. much cleaner.